### PR TITLE
UI lib updates 1

### PR DESCRIPTION
**Pull request by brentjk**
Update jQuery to 3.7.0
Update Bootstrap to 3.3.7
Update Ace Editor to v1.5.0

### CODE REVIEW

**Comment by brentjk** [comment]
Passes sanity check. will run all pages and will run sample process.

**Comment by brentjk** [comment]
Branch definitely is working but its the testing that I'm currently troubleshooting.

**Comment by brentjk** [comment]
"WARNING: Unable to find an exact match for CDP version 117, so returning the closest version found: 115"

**Comment by brentjk** [comment]
selinum v4.13.0 - is the first compatible with 117

**Comment by brentjk** [comment]
Running a test branch. Looks like some of the xml files are not coming through as intended and some of the table data is not seen during a page search.

**Comment by brentjk** [comment]
Clearly displaying log correctly but now showing 'graphite' text for the LogsTestIT-runOutputTest. Need to better understand graphite to see how its working under the hood. Are we missing some log set or other needed element.

**Comment by brentjk** [comment]
No errors for the :"Tests run: 1, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 48.321 s <<< FAILURE! - in jpl.cws.test.integration.ui.InitiatorsTestIT" going to need to step through to actually see what's happening. Looks fine on personal and server check.

**Comment by brentjk** [comment]
**note \\@test to skip test

**Comment by brentjk** [comment]
Ok only 3 failures left:

Have some "temp" prints that need cleaned up before can be merged. 

1.) Graphite not found on page
Part of test: ------ START LogsTestIT:runOutputTest ------

	INSTALLER: 2023-10-10T03:13:54,232 ERROR [                main]       jpl.cws.test.WebTestUtil( 155) - text 'Graphite' NOT FOUND on page
java.lang.AssertionError: text 'Graphite' NOT FOUND on page

This has something to do with graphite logging. not sure what yet. 


2.) runStartInitiatorTest

suspect its loosing object within a table

INSTALLER: 2023-10-10T03:18:33,947 INFO  [                main] ntegration.ui.InitiatorsTestIT( 172) - No stats for this process
INSTALLER: 2023-10-10T03:18:33,947 INFO  [                main] ntegration.ui.InitiatorsTestIT( 185) - ------ END


3.) runResultsTest
This is loosing object modifed by js. It saves the element 'history button' but crashes after calling it later.

**Comment by wcgunter** [comment]
For #1 I feel like the jQuery migration updated a function that is used to load/manipulate the data. I'm re-adding the migrate plugin and printing the console log to see if I can get any clues.

File: cws-test/src/test/java/jpl/cws/test/WebTestUtil.java [hunk 1]
```diff
@@ -151,6 +151,7 @@ protected boolean findOnPage(String text) {
 		}
 
 		if (!found) {
+			log.info("TEMP: " + driver.getPageSource());
```
Comment (jamesfwood) [inline cws-test/src/test/java/jpl/cws/test/WebTestUtil.java:154]: @wcgunter is this needed?

**Review by jamesfwood — APPROVED** [review]

**Pull request merged**